core/src/io/anuke/mindustry/ui/dialogs/SettingsMenuDialog.java

```diff
@@ -252,6 +252,7 @@ public void add(SettingsTable table){
         });
         graphics.sliderPref("fpscap", 240, 15, 245, 5, s -> (s > 240 ? Core.bundle.get("setting.fpscap.none") : Core.bundle.format("setting.fpscap.text", s)));
         graphics.sliderPref("chatopacity", 100, 0, 100, 5, s -> s + "%");
+        graphics.sliderPref("lasersopacity", 50, 0, 100, 5, s -> s + "%");
```

Comment:
If you're going to add a slider for laser opacity, then it would be a good idea to remove the setting for making power lasers visible, as it is no longer required. As an optimization, one could do `if(Core.settings.getInt("lasersopacity") == 0) return;` to prevent drawing anything when power lasers are hidden.

Reply:
Yeah, why not. I was also considering caching the setting value between each laser render to prevent queries to settings map. But it's not easy if we want to see the change without restarting the game.